### DISCUSSION TITLE
Health endpoint

### DISCUSSION THREAD

**Discussion by TheUdK** [Lissy93/dashy#696] [Help and Support]
Hi all,

Does dashy have a healthz endpoint that can be used for monitoring?

**Comment by lissy93** [comment]
Yes, 
AFK atm, but I think it's `yarn health-check`

**Comment by TheUdK** [comment]
im looking for something like this https://docs.gitlab.com/ee/user/admin_area/monitoring/health_check.html

remote callable url that provides the status of the services of dashy. so monitoring does not have to load the entire webpage content on every pass.
